src/rdkafka.c

```diff
@@ -5077,6 +5077,38 @@ rd_kafka_Uuid_t *rd_kafka_Uuid_copy(const rd_kafka_Uuid_t *uuid) {
         return copy_uuid;
 }
 
+/**
+ * Returns a new non cryptographically secure UUIDv4 (random).
+ *
+ * @return A UUIDv4.
+ *
+ * @remark Must be freed after use using rd_kafka_Uuid_destroy().
+ */
+rd_kafka_Uuid_t rd_kafka_Uuid_random() {
```

Comment:
Move this to rdkafka_proto.h where all the other internal functions related to Uuid are present

Reply:
that is just a header we cannot move all the implementation there, we can refactor to its own file